x-pack/plugins/maps/public/inspector/vector_tile_adapter/components/tile_request_tab.tsx

```diff
@@ -90,7 +90,7 @@ export function TileRequestTab(props: Props) {
           </EuiFlexGroup>
         </EuiFlexItem>
         <EuiFlexItem grow={true}>
-          <CodeEditor
+          <CodeEditor // TODO Unable to find context being provided. Need help from CodeOwners
```

Comment:
You can import a getter function to access theme service in maps plugin `import { getTheme } from '../../../kibana_services';`

Reply:
I'm having trouble navigating to this CodeEditor, can you help me figure out where it is?

Reply:
* install sample web logs data set
* create new map
* Set time range to last 24 hours
* Click "Add layer"
* Select "Documents"
* Select sample logs data view
* You should see dots on a map now
    <img width="500" alt="Screen Shot 2023-06-26 at 1 26 12 PM" src="https://github.com/elastic/kibana/assets/373691/5d0b8fba-594a-4e03-98b2-e4eacd3c9397">
* Click "Inspect". The inspector will open for "Vector tiles" and you can see the request body in the code editor
    <img width="500" alt="Screen Shot 2023-06-26 at 1 27 22 PM" src="https://github.com/elastic/kibana/assets/373691/5b129f1c-fbb3-42a6-97a9-731a34cd5dbc">

Reply:
++ thank you!! That `CodeEditor` looks like it has the theme being provided (I am not super familiar with that plugin):
<img width="1274" alt="Screenshot 2023-06-26 at 3 33 20 PM" src="https://github.com/elastic/kibana/assets/21210601/cbf7b267-f93c-47b7-9ebf-449dedfe625c">